project/Dependencies.scala

```diff
@@ -77,9 +74,6 @@ object Dependencies {
   val nettyCodec      = "io.netty"                % "netty-codec"                             % V.nettyCodec
   val nettyCodecHttp  = "io.netty"                % "netty-codec-http"                        % V.nettyCodec
   val nettyCodecHttp2 = "io.netty"                % "netty-codec-http2"                       % V.nettyCodec
-  val fastjson        = "com.alibaba"             % "fastjson"                                % V.fastjson
```

Comment:
Probably easiest to merge the three commits removing these three overrides into one; rather than migrate each change to the 'correct' commit.